### PR TITLE
Add SCOPES.md like the other SCOPES.md being added. And remove non-standard scopes.md

### PR DESCRIPTION
**Pull request by Callek**
@JohanLorenzo I'm not 100% sure my prose is correct or that it makes sense in this location.

In particular the 'esr' channel (which is allowed per script) isn't restricted and isn't mentioned in existing scopes.md

@escapewindow also interested in your feedback

### CODE REVIEW

File: SCOPES.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+* `{scope_prefix}:firefox:candidate`
+  * Uploads to the *candidate* snap track.
+    * Uses branding "Firefox"
+  * **Conflicts**: with any other `{scope_prefix}:firefox:*`
+  * **Branch Restrictions**:
+    * `release`
+
+* `{scope_prefix}:firefox:esr`
+  * Uploads to the ???.
```
Comment (JohanLorenzo) [inline SCOPES.md]: `to the esr (esr/stable) snap track.`. On https://snapcraft.io/firefox, click on "Install..." and see the esr track in the dropdown menu.

File: SCOPES.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  * **Does not seem to be used**
+  * **Conflicts**: with any other `{scope_prefix}:firefox:*`
+  * **Branch Restrictions**: `None`
```
Comment (JohanLorenzo) [inline SCOPES.md]: I should restrict this scope to esr60.

File: SCOPES.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  * **Conflicts**: with any other `{scope_prefix}:firefox:*`
+  * **Branch Restrictions**:
+    * `all-staging-branches`
+
+
+# Snap Track Details
+Snap store allows a product to have 4 different channels (edge, beta, candidate, release).
+Tracks are used by end-users when they want to enroll in a less stable version of Firefox.
+
+Pushapk accepts a subset of what the [Snap store allows](https://docs.snapcraft.io/reference/channels#risk-levels-meaning)
```
Comment (JohanLorenzo) [inline SCOPES.md]: Nit: pushsnap

**Comment by escapewindow** [comment]
+1 to Johan's comments, thank you!

File: SCOPES.md (hunk 2 above)
Comment (Callek) [reply SCOPES.md]: @JohanLorenzo reminder to do a PR for scriptworker to do that restriction.

**Comment by Callek** [comment]
@JohanLorenzo can you merge and/or grant releng write access?

**Comment by JohanLorenzo** [comment]
I don't have access to the settings options. @catlee, it this something you can help with? In the meantime, I merged this PR.

**Pull request merged**